src/content/language/mixins.md

```diff
@@ -60,30 +61,104 @@ mixin Musical {
+but can't define those members itself (because mixins can't use constructor
+parameters to instantiate their own fields).
+
+The following sections cover different strategies for ensuring any subclass
+of a mixin defines any members the mixin's behavior depends on. 
+
+### Define abstract members in the mixin
+
+By declaring the mixin as abstract, you force any type that uses
+it to define the abstract method upon which its behavior depends. 
+
+```dart
+abstract mixin Musician {
+  void playInstrument(String instrumentName); // Abstract method.
+
+  void playPiano() {
+    playInstrument('Piano');
+  }
+  void playFlute() {
+    playInstrument('Flute');
+  }
+}
+
+class Virtuoso with Musician { 
+  void playInstrument(String instrumentName) { // Subclass must define.
+    print('Plays the $instrumentName beautifully');
+  }  
+} 
+```
+
+#### Access state in the mixin's subclass
+
+Declaring abstract memebers also allows you to access state on the subclass
+of a mixin, by calling getters which are defined as abstract on the mixin:
+
+```dart
+/// Can be applied to any type with a [name] property and provides an
+/// implementation of [hashCode] and operator `==` in terms of it.
+mixin NameIdentity {
+  String get name;
+
+  int get hashCode => name.hashCode;
+  bool operator ==(other) => other is NameIdentity && name == other.name;
+}
+
+class Person with NameIdentity {
+  final String name;
+
+  Person(this.name);
+}
+```
+
+### Implement an interface
+
+Similar to declaring the mixin abstract, putting an `implements` clause on the
+mixin while not actually implementing the interface will also ensure any member
+dependencies are defined for the mixin.
+
+```dart
+?
+```
+
+### Use the `on` clause to declare a superclass
+
+The `on` clause exists to define the type that `super` calls are resolved against.
+So, you should only use it if you need to have a `super` call inside a mixin. 
+
+The `on` clause forces any class that uses a mixin to also be a subclass
+of the type in the `on` clause.
+If the mixin depends on members in the superclass,
+this ensures those members are available where the mixin is used:
 
-<?code-excerpt "misc/lib/language_tour/classes/orchestra.dart (mixin-on)" plaster="none" replace="/on Musician2/[!on Musician!]/g"?>
 ```dart
 class Musician {
-  // ...
+  musicianMethod() {
+    print('Playing music!');
+  }
 }
 
 mixin MusicalPerformer [!on Musician!] {
-  // ...
+  perfomerMethod() {
+    print('Performing music!');
+    super.musicianMethod();
+  }
 }
-class SingerDancer extends Musician with MusicalPerformer {
-  // ...
+
+class SingerDancer extends Musician with MusicalPerformer { }
+
+main() {
+  SingerDance().performerMethod();
 }
 ```
 
-In the preceding code,
-only classes that extend or implement the `Musician` class
-can use the mixin `MusicalPerformer`.
-Because `SingerDancer` extends `Musician`,
+In this example, only classes that extend or implement the `Musician` class
```

Comment:
The "or implement" part isn't quite right. I can explain exactly why if you want, but the simplest fix is to just leave it out and only say "extend".